conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosGovernanceOperations.scala

```diff
@@ -119,8 +115,8 @@ object TezosGovernanceOperations extends LazyLogging {
     * @param ec needed to compose concurrent operations
     * @return aggregated data
     */
-  private def aggregateData(db: Database)(
-      proposalsBlocks: Map[Block, ProtocolId],
+  def aggregateData(db: Database)(
+      proposalsBlocks: Map[Block, Option[ProtocolId]],
```

Comment:
We probably have a bug here too. `proposalsBlocks` is ultimately derived from `TezosGovernanceOperations`, line 84:

```scala
activeProposals <- nodeOperator.getProposals(blocksInActiveVotingPeriod)
```

This only picks up blocks with active proposals. This excludes blocks from the proposal period where there is no one active proposal. When you run your local testing, you will not see any `governance` records from the proposals periods. Once you fix the bug, you will start seeing valid records for the proposals periods. Please use this as a validation of your code.

Reply:
I have to disagree here: `activeProposals` collects by block any active proposal id, yet immediately after we associate those _optionally_ , with the original block list, at line 87.
That is, we associate each block with a proposal id, if there's one, otherwise the optional value is empty.

It's a sparse associative array, there are holes in the map values, corresponding to the proposal periods

Reply:
In fact I did test locally and saw those entries being saved, where previously there were none.

I can check once more, but I have logs that still shows

```
17:19:52.557 [lorre-tezos-indexer-akka.actor.default-dispatcher-40] INFO  t.c.c.i.t.TezosGovernanceOperations$ - Searching for governance data in voting period...
17:19:52.558 [lorre-tezos-indexer-akka.actor.default-dispatcher-40] INFO  t.c.c.i.t.TezosGovernanceOperations$ - There are no blocks related to some voting - excluding proposals.
17:19:53.502 [lorre-tezos-indexer-akka.actor.default-dispatcher-40] INFO  t.c.c.i.t.TezosDatabaseOperations$   - Writing 1000 governance rows into database...
```

Which attest that 1000 rows were written, even if no active voting period blocks were found